### PR TITLE
Thunderloop wait wifi

### PR DESCRIPTION
**Pull request by Bvasilchikov**
### Description

The current configuration of the robot doesn't wait for a rout-able network connection before starting up thunderloop. This causes an issue where thunderloop crashes when trying to instantiate the network service.

This PR adds the `network-online.target` flag to the thunderloop and display systemd services running on the robot. It also adds a command that's executed in deployment to enable the native service that checks for a network connection (`NetworkManager-wait-online.service`). After all the robots have this service enabled, it can be removed from deploy moved to `setup_nano.yml`

### Testing Done

Local testing by adding the `network-online.wait` target, and restarting the robot a bunch of times to make sure it never boots up with a crashed thunderloop. 

Need to test `deploy_nano.yml` enables network waiting service.

### Review Checklist

**_It is the reviewers responsibility to also make sure every item here has been covered_**

- [ ] **Function & Class comments**: All function definitions (usually in the `.h` file) should have a javadoc style comment at the start of them. For examples, see the functions defined in `thunderbots/software/geom`. Similarly, all classes should have an associated Javadoc comment explaining the purpose of the class.
- [ ] **Remove all commented out code**
- [ ] **Remove extra print statements**: for example, those just used for testing
- [ ] **Resolve all TODO's**: All `TODO` (or similar) statements should either be completed or associated with a github issue

### CODE REVIEW

File: src/software/jetson_nano/linux_configs/systemd/display.service
Comment (nimazareian) [inline src/software/jetson_nano/linux_configs/systemd/display.service:1]: I thought `display.service` was removed. @LiCody Do we still need to keep it?

File: src/software/jetson_nano/ansible/playbooks/deploy_nano.yml [hunk 1]
```diff
@@ -101,3 +101,8 @@
             loop_var: service_name
           tags: always
           failed_when: false
+        - name: Enable NetworkManager-wait-online.service
+          become: true
+          become_method: sudo
+          command: 'sudo systemctl enable NetworkManager-wait-online.service'
+          tags: always
```
Comment (nimazareian) [inline src/software/jetson_nano/ansible/playbooks/deploy_nano.yml]: I think we should also be adding these changes to `setup_nano.yml`

**Review by itsarune — COMMENTED** [review]
looks good, pending nima's comments

File: src/software/jetson_nano/ansible/playbooks/setup_nano.yml [hunk 2]
```diff
@@ -215,6 +215,16 @@
           tags:
             - enableSystemd
 
+        - name: Enable NetworkManager-wait-online.service
+          become: true
+          become_method: sudo
+          ansible.builtin.syytemd:
```
Comment (sauravbanna) [inline src/software/jetson_nano/ansible/playbooks/setup_nano.yml]: is this right?

File: src/software/jetson_nano/ansible/playbooks/setup_nano.yml [hunk 3]
```diff
@@ -215,6 +215,16 @@
...
+            name: NetworkManager-wait-online
+            enabled: yes
+            masked: no
+            daemon_reload: tes
```
Comment (sauravbanna) [inline src/software/jetson_nano/ansible/playbooks/setup_nano.yml]: this should be 'yes' i think

**Review by sauravbanna — DISMISSED** [review]

File: src/software/jetson_nano/linux_configs/systemd/display.service
Comment (LiCody) [reply src/software/jetson_nano/linux_configs/systemd/display.service:1]: we dont need it currently but dont mind leaving if we end up using a display next yr

**Review by LiCody — APPROVED** [review]
LGTM

**Review by nimazareian — APPROVED** [review]

**Pull request merged**